docs/components/copy.md

```diff
@@ -0,0 +1,183 @@
+# Copy
+
+KCopy is a component that displays provided text and allows a user to copy it to their clipboard.
+
+<KCopy :text="text" />
+
+```html
+<KCopy :text="text" />
+```
+
+## Props
+
+### text
+
+The copyable text.
+
+### badge
+
+Whether or not to display as a badge. Defaults to `false`.
+
+<KCopy
+  badge
+  truncate
+  :text="text"
+/>
+
+```html
+<KCopy
+  badge
+  truncate
+  :text="text"
+/>
+```
+
+### badgeText
+
+Text displayed before the copyable text when `badge` is true.
+
+<KCopy
+  badge
+  badge-text="Id"
+  :text="text"
+/>
+
+```html
+<KCopy
+  badge
+  badge-text="Id"
+  :text="text"
+/>
+```
+
+### truncate
+
+Whether or not the text should be truncated. Defaults to `false`.
+
+<KCopy
+  truncate
+  :text="text"
+/>
+
+```html
+<KCopy
+  truncate
+  :text="text"
+/>
+```
+
+### truncationLimit
+
+Number of characters to truncate at. Defaults to `8`.
+
+<KCopy
+  truncate
+  :text="text"
+  :truncation-limit="15"
+/>
+
+```html
+<KCopy
+  truncate
+  :text="text"
+  :truncation-limit="15"
+/>
+```
+
+### copyTooltip
+
+Tooltip text displayed on hover copy button. 
+If the `badgeText` prop has a value, then the copy tooltip text is `Copy {badgeText}`; otherwise, `Copy`.
```

Comment:
this doesn't mention that you're stripping certain characters from the `badgeText` for the tooltip?

Reply:
Added.
<img width="768" alt="image 1647" src="https://github.com/Kong/kongponents/assets/87779967/82c94888-9711-4cef-91d5-6300388e3f3c">